scripts/linters/general_purpose_linter_test.py

```diff
@@ -86,6 +86,10 @@
     LINTER_TESTS_DIR, 'invalid_http_client_used.ts')
 
 # PY filepaths.
+INVALID_PUT_ASYNC_FILEPATH = os.path.join(
+    LINTER_TESTS_DIR, 'invalid_put_async.py')
+INVALID_PUT_MULTI_ASYNC_FILEPATH = os.path.join(
+    LINTER_TESTS_DIR, 'invalid_put_multi_async.py')
```

Comment:
This file is not commited?

Reply:
I am sorry that file was not added to the commit. I have added it now. Done.